utils/ssl_renewel_util/README.md

```diff
@@ -0,0 +1,82 @@
+# SSL Certificate Renewal Script for Aws
+Note: If you are not using Aws as your Dns provider then this script won't work. You'll have to manually renewel the certificates, In case if you are using any other Dns provider then check if certbot supports any dns plugin for your Dns provider and then just update the certbot command then the script should work fine.
+
+This script is used for the renewal of SSL certificates using Let's Encrypt's `certbot`. It moves old certificates to a backup directory with an expiry date and renew the ssl certificates and Update cert location.
+
+## Prerequisites
+
+- **Certbot**: Ensure `certbot` is installed and configured on your system.
+- **AWS Route 53**: The script uses Route 53 for DNS challenges.
+- **Sudo Privileges**: Required to move directories and restart services.
+- **Nginx**: Ensure Nginx is installed and configured to use Let's Encrypt certificates.
+
+## Installation
+
+1. **Install Certbot**: Follow the instructions [here](https://certbot.eff.org/instructions) to install `certbot` for your nginx server and operating system.
```

Comment:
**Install Certbot**: Follow the instructions [here](https://certbot.eff.org/instructions) to install `certbot` for your nginx server

Reply:
updated